src/scripting_api/initialization.js

```diff
@@ -52,6 +52,10 @@ function initSandbox(data, extra, out) {
       out[name] = aform[name].bind(aform);
     }
   }
+
+  if (testMode) {
+    out._app = app;
+  }
```

Comment:
Do you perhaps want to *explicitly* prevent this from being set *unless* testing is running?
If so, the following prevent the parameter being respected in "regular" builds:
```suggestion
  if (
    (typeof PDFJSDev === "undefined" ||
      PDFJSDev.test("!PRODUCTION || TESTING")) &&
    testMode
  ) {
    out._app = app;
  }
```